pages/linux/rtcwake.md

```diff
@@ -0,0 +1,36 @@
+# rtcwake
+
+> Enter a system sleep state until specified wakeup time relative to your bios clock.
+> Commands need to be run as superuser since shutdowns are involved.
+
+- Check your hardware clock info and exit:
+
+`rtcwake -m show -v`
+
+- Show if an alarm is set:
+
+`rtcwake -m show`
+
+- Disable a previous set alarm:
+
+`rtc -m disable`
+
+- Suspend to ram and wakeup after 10 seconds:
+
+`sudo rtcwake -m mem -s {{10}}`
+
+- Suspend to disk (higher power saving) and wakeup 10 minutes later:
+
+`sudo rtcwake -m disk --date +{{15}}m`
+
+- Suspend to disk and wakeup 1 hour later and play some music with mpv:
```

Comment:
Why is this example necessary ? I don't see it adding anything new except using hour as a unit.

Reply:
It's a useful usage example imo... for example, I use the command mainly as a wake-up alarm.

Reply:
This is showing composition of one command with another command. You are combining the wakeup action with playing music after it to make it behave like an alarm. Strictly speaking, this behavior is outside the scope of a tldr page for rtcwake, which should just be focused on explaining the command usage. A user can easily construct an example like this if he/she understands the command well enough (which should be our primary aim).